build_tools/release_automation.py

```diff
@@ -136,15 +206,16 @@ def generate_zenodo(sasview_data, zenodo_api_key):
                                                             "and attach to Zenodo release record.\n- Publish Zenodo record")
     return newDOI
 
+
 version_template = \
-"""
-__version__ = "%s"
+"""__version__ = "%s"
 __release_date__ = "%i"
 __build__ = "GIT_COMMIT"
```

Comment:
This line seems superfluous and outdated?  The original concept was that the installer would have a build number (as many software packages do) which would be the commit hash, but that only worked briefly. Does "GIT_COMMIT" have any utility? Of course, even if it is pointless, it may be something to remove after the release in which case we should ticket it.

Reply:
Good question. Probably not necessary, but I agree we shouldn't hold up a beta release for this.